dnf-behave-tests/dnf/microdnf/reinstall5.feature

```diff
@@ -0,0 +1,18 @@
+@no_installroot
+Feature: Reinstall
+
+
+Scenario: Try to reinstall a pkg if repo not available
+  Given I use repository "simple-base"
+    And I successfully execute microdnf with args "install labirinto"
+   When I configure a new repository "non-existend" with
```

Comment:
I know it doesn't affect anything, but can you rename it to "non-existent" (with "t" at the end)? It's also in the `microdnf/makecache.feature`.

Reply:
Sure, updated.